build.sbt

```diff
@@ -74,7 +74,9 @@ val root = project("scala-native-bindgen")
         releaseStepCommandAndRemaining("verify"),
         setReleaseVersion,
         tagRelease,
-        releaseStepCommandAndRemaining("^publish"),
+        releaseStepCommandAndRemaining("+bindings/publish"),
+        releaseStepCommandAndRemaining("+tools/publish"),
+        releaseStepCommandAndRemaining("^sbt-scala-native-bindgen/publish"),
```

Comment:
What is the difference between `^`, `+` and no prefix? I could not find the answer in google

Reply:
`^` means for all sbt versions configured in `crossSbtVersions`.

`+` means for all scala versions configured in `crossScalaVersions`.

No prefix means use current sbt and scala version.